awesomerc.lua

```diff
@@ -434,6 +434,11 @@ client.connect_signal("request::default_keybindings", function()
                 c:raise()
             end ,
             {description = "(un)maximize horizontally", group = "client"}),
+        awful.key({ modkey,           }, "z",
```

Comment:
Still here ;)

Reply:
Oh my God hahahahahaahahahah

THANK YOU for combing this line by line, these last commits have been a trainwreck.

Reply:
Line 441: `description = "test keyname", group = "shit"`. Real classy, myself. _That's_ code ready for release.